liota/core/package_manager.py

```diff
@@ -40,7 +40,8 @@
 from Queue import Queue
 from time import sleep
 from abc import ABCMeta, abstractmethod
-from liota.lib.utilities.utility import read_liota_config, sha1sum, validate_named_pipe
+
```

Comment:
Remove the extra line

Reply:
This is a revert of a commit. I don't want to introduce changes in a revert.

Reply:
They are not changes, it is code formatting as this is getting introduced as part of the new commit.